main/manager.c

```diff
@@ -712,6 +712,9 @@
 			<parameter name="OtherChannelId">
 				<para>Channel UniqueId to be set on the second local channel.</para>
 			</parameter>
+            <parameter name="PreDialGoSub">
```

Comment:
The indentation is wrong on this as well.

Reply:
OK, solved. Thanks.